src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -51,7 +51,7 @@ public class HostJsonFileConfigurationProvider : ConfigurationProvider
             {
                 "version", "functionTimeout", "functions", "http", "watchDirectories", "queues", "serviceBus",
                 "eventHub", "singleton", "logging", "aggregator", "healthMonitor", "extensionBundle", "managedDependencies",
-                "httpWorker"
+                "customHandler"
```

Comment:
>customHandler [](start = 17, length = 13)

aren't we supporting both for sometime  ?

Reply:
You are right. Will be good to leave this here. No need to add metric but have having host.json will help us differentiate. Will update